### PR TITLE
feat: Added curl command syntax in config for  chat-application 

### PR DESCRIPTION
**Pull request by ChoudharyHarish**
Added Curl request syntax for chatbot in config modal.

### CODE REVIEW

File: frontend/src/screens/dashboard/docsqa/Applications/index.tsx [hunk 1]
```diff
@@ -136,13 +163,38 @@ const Applications = () => {
...
+                    )}
                   </ul>
                 </>
               )}
+
+              <button
+                onClick={() => setShowCurl(!showCurl)}
+                className="my-2 text-sm p-2 bg-[#6366F1] text-white rounded"
+              >
+                {showCurl ? 'Hide' : 'Show Curl Command'}
```
Comment (sayan-truefoundry) [inline frontend/src/screens/dashboard/docsqa/Applications/index.tsx:182]: i would suggest lets keep 'Curl Command' string for both conditions like :
```suggestion
                {showCurl ? 'Hide' : 'Show'} Curl Command
```

File: frontend/src/screens/dashboard/docsqa/Applications/index.tsx [hunk 2]
```diff
@@ -136,13 +163,38 @@ const Applications = () => {
...
+              </button>
+
+              {showCurl && (
+                <div className="flex flex-col gap-2 mt-2">
+                  <h1>curl Request Example : </h1>
```
Comment (sayan-truefoundry) [inline frontend/src/screens/dashboard/docsqa/Applications/index.tsx:187]: lets update this header to :
`cURL Request Example`

File: frontend/src/screens/dashboard/docsqa/index.tsx [hunk 3]
```diff
@@ -61,7 +61,7 @@ const DocsQA = () => {
   const [modelConfig, setModelConfig] = useState(defaultModelConfig)
   const [retrieverConfig, setRetrieverConfig] = useState(defaultRetrieverConfig)
   const [promptTemplate, setPromptTemplate] = useState(defaultPrompt)
-  const [isStreamEnabled, setIsStreamEnabled] = useState(true)
+  // const [isStreamEnabled, setIsStreamEnabled] = useState(true)
```
Comment (sayan-truefoundry) [inline frontend/src/screens/dashboard/docsqa/index.tsx:64]: are we removing streaming toggle option? if yes, then remove this state completely

File: frontend/src/screens/dashboard/docsqa/Applications/index.tsx (hunk 1 above)
Comment (ChoudharyHarish) [reply frontend/src/screens/dashboard/docsqa/Applications/index.tsx:182]: I have changes the header, stream option completely, show/hide text, and clipboard icon issue , and made close button fixed in button Review it once @sayan-truefoundry  and ping me if any other fix is needed

**Review by sayan-truefoundry — APPROVED** [review]

**Pull request merged**